src/Testcontainers.Papercut/PapercutBuilder.cs

```diff
@@ -0,0 +1,71 @@
+using System.Net;
+
+namespace Testcontainers.Papercut;
+
+/// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+[PublicAPI]
+public sealed class PapercutBuilder : ContainerBuilder<PapercutBuilder, PapercutContainer, PapercutConfiguration>
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Testcontainers.PapercutBuilder" /> class.
+    /// </summary>
+    public PapercutBuilder()
+        : this(new PapercutConfiguration())
+    {
+        DockerResourceConfiguration = Init().DockerResourceConfiguration;
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Testcontainers.PapercutBuilder" /> class.
+    /// </summary>
+    /// <param name="resourceConfiguration">The Docker resource configuration.</param>
+    private PapercutBuilder(PapercutConfiguration resourceConfiguration)
+        : base(resourceConfiguration)
+    {
+        DockerResourceConfiguration = resourceConfiguration;
+    }
+
+    /// <inheritdoc />
+    protected override PapercutConfiguration DockerResourceConfiguration { get; }
+
+    /// <inheritdoc />
+    public override PapercutContainer Build()
+    {
+        Validate();
+        return new PapercutContainer(DockerResourceConfiguration, TestcontainersSettings.Logger);
+    }
+
+    /// <inheritdoc />
+    protected override PapercutBuilder Init()
+    {
+        return base.Init()
+            .WithImage("jijiechen/papercut:latest")
```

Comment:
Please use a const and pin the image version, as is done with all the other modules.

Reply:
Moved it to a const not sure how to pin the version as it's not tagged in the docker hub had a look at how the other packages are doing this and could see that some of the packages e.g. Testcontainers.CosmosDb are using :latest

I'm reasonably new to docker stuff though so if there is a way to do this please let me know I'm happy to add it in

Reply:
> not sure how to pin the version as it's not tagged

👍 Okay, then `latest` is fine. I didn't check the image tags. Thanks for checking.